test/go.mod

```diff
@@ -2,39 +2,22 @@ module github.com/mesosphere/kubeaddons-base-addons/test
 
 go 1.15
 
-replace k8s.io/client-go => k8s.io/client-go v0.19.0
+replace (
+	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce // locked to this version to avoid upgrades from kind that would drop our volume provisioning code. TODO: we should update/remove our volume provisioning code.
+	google.golang.org/grpc => google.golang.org/grpc v1.26.0 // held back to accomodate controller-runtime which has not updated for v1.3x.x+ at the time of writing
```

Comment:
I'm not sure I understand why this needs to be replaced. What is the dependency causing this problem? I can't see any requirement of grpc in [controller-runtime's go.mod](https://github.com/kubernetes-sigs/controller-runtime/blob/master/go.mod). I'd mainly like to understand and clarify on this replacement.

Reply:
I may have mistaken which module was doing this, I'll review one second.

Reply:
This was a mistake, I worked around this already and this can be removed now :+1: